src/incremental/__init__.py

```diff
@@ -532,4 +532,8 @@ def _extract_tool_incremental(data):  # type: (Dict[str, object]) -> Optional[Di
 from ._version import __version__  # noqa: E402
 
 
+def _setuptools_version():  # type: () -> str
+    return __version__.public()  # pragma: no cover
```

Comment:
This is the hack I tried to remove in bd4d87bb1ec94a3a38f41980b3580ea97f467bb3. It turns out that only worked because an older version of Incremental was already installed in the build environment under Tox.

A previous Incremental is not installed when using `build` to generate the wheel, as this PR changes the GitHub Actions workflow to do.